models/user.js

```diff
@@ -2,6 +2,7 @@
 // https://github.com/brianc/node-pg-query
 
 var db = require('../db');
+var bitcore = require('bitcore');
 var query = db.query;
```

Comment:
Had been using that for FastBets (just wasn't sure when you asked). We might have to move some code I wrote from the models to the controllers. Will get some rest now and join you coding when I get up :-)